src/BlockTypes/AllProducts.php

```diff
@@ -46,18 +46,18 @@ protected function register_block_type_assets() {
 		parent::register_block_type_assets();
 		$this->register_chunk_translations(
 			[
-				'atomic-block-components/price',
-				'atomic-block-components/image',
-				'atomic-block-components/title',
-				'atomic-block-components/rating',
-				'atomic-block-components/button',
-				'atomic-block-components/summary',
-				'atomic-block-components/sale-badge',
-				'atomic-block-components/sku',
-				'atomic-block-components/category-list',
-				'atomic-block-components/tag-list',
-				'atomic-block-components/stock-indicator',
-				'atomic-block-components/add-to-cart',
+				'product-price',
+				'product-image',
+				'product-title',
+				'product-rating',
+				'product-button',
+				'product-summary',
+				'product-sale-badge',
+				'product-sku',
+				'product-category-list',
+				'product-tag-list',
+				'product-stock-indicator',
+				'product-add-to-cart',
 			]
 		);
 	}
```

Comment:
While reviewing this PR we've noticed that changes to the names of the packages unfortunately affected ability to read translation files for the blocks. 

The change to how atomic blocks were registered for translation chunks was controlled via the `register_block_type_assets` method on the `All Products`. After the blocks package chunk name changed from `atomic-block-components/price` pattern to `product-price` pattern the translations are no longer recognised as they were referenced in the .po files via their old chunk name. 

Before this can be merged we need to learn if we can update the translation files easily to make it work or if we need to adopt the whole code to support custom names/paths for atomic blocks family with old names until the translation files can be migrated to new names that would highlight these blocks moving to be first party blocks.

Apart of that now that all atomic blocks have their own php registration we should have them register the translation chunk by themselves rather than have All Blocks control this responsibility.

Reply:
Also we should probably cover i18n with E2E - we've caught this one by chance 😨